packages/eslint-config-react/index.js

```diff
@@ -206,6 +231,18 @@ module.exports = {
     // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/void-dom-elements-no-children.md
     'react/void-dom-elements-no-children': 'error',
 
+    // Enforce boolean attributes notation in JSX
+    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md
+    'react/jsx-boolean-value': ['warn', 'never'],
+
+    // Enforce or disallow spaces inside of curly braces in JSX attributes and expressions.
+    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-child-element-spacing.md
+    'react/jsx-child-element-spacing': 'error',
```

Comment:
This should go into _./style.js_ 🙏

Reply:
I was thinking of putting it into styles. But realized this actually can really cause real issues. Thus I decided to move into the index file.

Reply:
Ok let's leave here. Out of curiosity, what issues could this cause? 🤔